src/editor/components/dialog.tsx

```diff
@@ -3,7 +3,7 @@ import { XIcon } from '@heroicons/react/outline'
 
 interface DialogProps {
   isOpen: boolean,
-  title: string
+  title?: string
```

Comment:
It is really strange if you want to parse a markdown file otherwise, because you'd need to manually extract the `<h1>` element and set the title to it. Also, the formatting would be very strange. Do you have a better solution?

Reply:
Why do you want to extract the h1 element? The title of the dialog can always stay the same, something like `Node Documentation`. You could even use the title of the node.